src/inputs/ObjectSearchCreateSearchInput.tsx

```diff
@@ -153,16 +165,26 @@ class ObjectSearchCreateSearchInput extends Component<IObjectSearchProps> {
       </Antd.Select.Option>
     );
   }
+  private onChange (selectedOption: any) {
+    if (selectedOption.key === ITEM_KEYS.ADD) {
+      this.props.onAddNew(this.search);
+    }
 
-  private onChange (value: any) {
-    const foundOption = this.options.find(option => option.id === value.key);
+    const foundOption = this.options.find(option => option.id === selectedOption.key);
     this.injected.onChange(toJS(foundOption));
   }
 
+  private onFocus () {
+    if (!this.hasOptions && !this.hasSearch) {
```

Comment:
This `onFocus` is searching with an empty string to get a list of all possible options on initial click?

Reply:
I believe so. @kbouchard ?

Reply:
ex-ac-tly

Focusing a "list" selection that would render no values at first was really confusing UX.

Reply:
Then I would name a boolean something like that so it's clearer to the next person who reads it!